tests/load/pipeline/test_scd2.py

```diff
@@ -0,0 +1,370 @@
+# timezone is removed from all datetime objects in these tests to simplify comparison
+
+import pytest
+from typing import List, Dict, Any
+from datetime import datetime, timezone  # noqa: I251
+
+import dlt
+from dlt.common.pipeline import LoadInfo
+from dlt.common.schema.typing import DEFAULT_VALIDITY_COLUMN_NAMES
+from dlt.common.normalizers.json.relational import DataItemNormalizer
+from dlt.common.normalizers.naming.snake_case import NamingConvention as SnakeCaseNamingConvention
+from dlt.destinations.sql_jobs import HIGH_TS
+
+from tests.pipeline.utils import assert_load_info
+from tests.load.pipeline.utils import (
+    destinations_configs,
+    DestinationTestConfiguration,
+    load_tables_to_dicts,
+)
+
+
+ACTIVE_TS = datetime.fromisoformat(HIGH_TS.isoformat()).replace(tzinfo=None)
+h = DataItemNormalizer.get_row_hash
```

Comment:
maybe alias this to `get_row_hash`?

Reply:
Changed in https://github.com/dlt-hub/dlt/pull/1168/commits/7726d98a5a412da45b6a6a1f9bc3ffb736b781cd